doc/installing.rst

```diff
@@ -522,7 +546,8 @@ GCC 12.x via ``homebrew``, you would get ``pypeit`` to use it by doing, for exam
 
 .. code-block:: console
 
-    CC=gcc-12 pip install pypeit
+    $ export CC=gcc-12 
```

Comment:
should be gcc-14 as of now. `/opt/homebrew/bin/gcc` is more general option for homebrew installs.

Reply:
I went with the general /opt/homebrew/bin/gcc